integration/kubernetes/confidential/sev.bats

```diff
@@ -0,0 +1,323 @@
+#!/usr/bin/env bats
+# Copyright 2022 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+export TEST_DIR
+export ENCRYPTION_KEY
+
+export KBS_DIR
+export KBS_DB_HOST
+export KBS_DB_USER="kbsuser"
+export KBS_DB_PW="kbspassword"
+export KBS_DB="simple_kbs"
+export KBS_DB_TYPE="mysql"
```

Comment:
When mysql CLI is installed in the host?

Reply:
Added `mysql-client` to install packages in `sev.bats`.